### DISCUSSION TITLE
Trying to use leeway model in a tsunami simulation

### DISCUSSION THREAD

**Discussion by alexvillarroel** [OpenDrift/opendrift#1215] [Q&A]
Hi, i'm newer in Opendrift, i'm trying to use leeway model in a tsunami simulation, but i dont know how i can import my data, it's a matrix for each x_sea_water_velocity and x_sea_water_velocity, i wanna reproduce something like that: [Leeway example](https://opendrift.github.io/gallery/example_leeway.html#leeway)
I know that for the example use readers from an .nc file, i had tried with others readers but i dont know what kind of reader to use

Leeway model use:

x_sea_water_velocity ✅ 
y_sea_water_velocity ✅ 
x_wind_velocity ⛔ ( for this i will use constant of 0 bc the magnitude order of energy in tsunami velocities and wind velocity makes the last one negligible)
y_wind_velocity ⛔ (same)

My idea is to transform my data into a netcdf, but i dont know the format

Any help is very useful to me.
Thanks!

**Comment by knutfrode** [comment]
Hi,

You can probably follow this example: https://opendrift.github.io/gallery/example_static_2d_current.html

If making a netCDF file instead, it has to be CF-compliant: https://cfconventions.org
The specification is long and detailed, easiest is to try to copy the structure of an existing file.
